src/player.cpp

```diff
@@ -1824,6 +1824,16 @@ bool player::is_immune_effect( const efftype_id &effect ) const
     return false;
 }
 
+bool player::drag_check() const
+{
+    if (get_str() + (get_per() / 3) + (get_dex() / 4) > rng(4, 15) || is_throw_immune()){
```

Comment:
Wouldn't the parameter need to be stored in the json, next to the attack name and frequency? So something like ``["GRAB", 7, 10]``? Can I just add another field and then call it?